modules/web/js/ballerina/diagram/views/compact/dimension-visitors/aborted-statement-dimension-calculator-visitor.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import SizingUtil from './../sizing-util';
+
+/**
+ * Dimension visitor class for aborted statement.
+ *
+ * @class AbortedStatementDimensionCalculatorVisitor
+ * */
+class AbortedStatementDimensionCalculatorVisitor {
+
+    /**
+     * Constructor for Aborted statement dimensions
+     * @param {object} options - options
+     */
+    constructor(options) {
+        this.sizingUtil = new SizingUtil(options);
```

Comment:
Initializing a util function is not good. Util should be static.

Reply:
This is the first phase of iterative migration of the diagram rendering logic to perspective based rendering. In the current approach, this particular util needs to be configured based on the designer (merge of default and the perspective's designer). With the current implementation, we need to initialize the util.